README.md

```diff
@@ -56,14 +56,32 @@ The operator can be deployed with Helm v3 chart in /charts as follows:
   * Disable persistence for basic testing. This can be done by adding `--set
     etcd.persistence.enabled=false` to the `helm install` command below.
 
-- Configure the LVM VG and LV names by setting the following values in a local
-  values file or by using `--set` with the `helm install` command below.
-
-    ```
-    operator.nodeSet.spec.lvmPoolVgName=drbdpool # <- Volume Group name of the thick storage pool
-    operator.nodeSet.spec.lvmThinPoolVgName=drbdpool # <- Volume Group name of the thin storage pool
-    operator.nodeSet.spec.lvmThinPoolLvName=thinpool # <- Logical Volume name of the thin pool
-    ```
+- Configure storage pools. By default, helm creates two pools (see [values.yaml]):
```

Comment:
I actually think we should change the default to not create any pools. It would be unusual that the user wants to have precisely the 2 pools defined in `values.yaml` and we're still in 0.x version range so we don't need to worry too much about making breaking changes.

Reply:
Updated the default values for helm. During that I also noticed that there was a second copy of the CRDs hidden in the "non-helm" deployment examples. I removed them from the examples and added instructions to install them by hand.